plugins/metrics/runtime/runtime.go

```diff
@@ -0,0 +1,89 @@
+package runtime
+
+import (
+	"context"
+	"time"
+
+	"go.opentelemetry.io/otel/api/metric"
+	"go.opentelemetry.io/otel/api/unit"
+	"golang.org/x/time/rate"
+
+	"github.com/open-telemetry/opentelemetry-go-contrib/plugins/metrics/runtime/internal/metrics"
+)
+
+var (
+	initialisation time.Time
+)
+
+func init() {
+	initialisation = time.Now()
+}
+
+// various runtime metrics have a certain overhead, and as such, we only measure them on a defined interval
+func Runtime(ctx context.Context, meter metric.Meter, interval time.Duration) context.CancelFunc {
+	ctx, cancel := context.WithCancel(ctx)
+
+	ctxTimeout, _ := context.WithTimeout(ctx, time.Second)
+	var m, p metrics.Metrics
+	m, _ = metrics.Measure(ctxTimeout)
+
+	if interval == 0 {
+		interval = time.Second
+	}
+
+	go func() {
+		limit := rate.NewLimiter(rate.Every(time.Second), 1)
+		for {
+			err := limit.Wait(ctx)
+
+			if err != nil {
+				break
+			}
+
+			ctxTimeout, _ := context.WithTimeout(ctx, time.Second)
+			// store the previous measurement for reporting deltas
+			p = m
```

Comment:
When we introduce the CumulativeObserver, storing the previous measurement won't be needed.

Reply:
Correction: This will become a SumObserver. But since this PR is ready now, and the new instruments are not through specification and haven't been implemented yet, this isfine.